src/highlevel_feature_extractor.h

```diff
@@ -1,3 +1,5 @@
+// -*-c++-*-
```

Comment:
Enables some editors to recognize it as C++ instead of C, since the file ending could be a C header file. It was already in some of the files, actually.